### DISCUSSION TITLE
希望加一个屏蔽付费视频的功能

### DISCUSSION THREAD

**Discussion by DD224ever** [the1812/Bilibili-Evolved#4567] [功能建议 / Ideas]
每次动态看到点进去就提示付费 很无语。
![image_54](https://github.com/the1812/Bilibili-Evolved/assets/62013258/367d9e0c-1d83-4870-9144-7b6b25ecf4d4)
![image_55](https://github.com/the1812/Bilibili-Evolved/assets/62013258/aa33fba1-0711-48d1-8088-b41951f404d7)

**Comment by Geeyun-JY3** [comment]
纯好奇，视频卡片左上角的这个`已访问`和`已观看xx%`是怎么实现的？
